ghost/core/core/frontend/helpers/ghost_head.js

```diff
@@ -86,7 +86,7 @@ function getSearchHelper(frontendKey) {
 function getWebmentionDiscoveryLink() {
     try {
         const siteUrl = urlUtils.getSiteUrl();
-        const webmentionUrl = new URL('webmention', siteUrl);
+        const webmentionUrl = new URL('webmentions/receive', siteUrl);
```

Comment:
@SimonBackx I think we need a trailing slash on the end of this because Ghost adds slashes, and it'll result in a 301 otherwise

Reply:
Never mind, my testing was wrong